cmd/flags/blockchain.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package flags
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const (
+	rpcURLFLag = "rpc"
+)
+
+var RPC string
+
+func AddRPCFlagToCmd(cmd *cobra.Command) {
```

Comment:
Let's add a validation function + testcases to this case to confirm only valid RPCs URLs are being passed in.

-We should validate the URL structure
-What are your thoughts on trying to connect to the API to make sure that it's actually able to serve the necessary use case

Reply:
there is a validation function for urls into pkg/prompts/ we may consider to move this into pkg/validation and share.
the validation function checks or not URL API liveness under request

Reply:
I think moving validation things around in the codebase is out of scope for this PR